src/components/sbb-checkbox/sbb-checkbox.e2e.ts

```diff
@@ -1,88 +1,83 @@
     });
   });
 
   describe('indeterminate', () => {
     it('should set indeterminate to false after checked', async () => {
-      page = await newE2EPage();
-      await page.setContent('<sbb-checkbox indeterminate>Label</sbb-checkbox>');
-      element = await page.find('sbb-checkbox');
-      await page.waitForChanges();
-
-      expect(await element.getProperty('checked')).toBe(false);
-      expect(await element.getProperty('indeterminate')).toBe(true);
-
-      await element.click();
-      await page.waitForChanges();
-
-      expect(await element.getProperty('checked')).toBe(true);
-      expect(await element.getProperty('indeterminate')).toBeFalsy();
+      element.setAttribute('indeterminate', 'true');
+      await waitForLitRender(element);
+      expect(element.checked).to.be.false;
+      expect(element.indeterminate).to.be.true;
+
+      element.click();
+      await waitForLitRender(element);
+      expect(element.checked).to.be.true;
+      expect(element.indeterminate).to.be.false;
     });
 
     it('should update indeterminate state of input', async () => {
-      await page.waitForChanges();
+      expect(element.indeterminate).to.be.false;
 
-      expect(await element.getProperty('indeterminate')).toBeFalsy();
+      element.indeterminate = true;
+      await waitForLitRender(element);
 
-      element.setProperty('indeterminate', true);
-      await page.waitForChanges();
-
-      expect(await element.getProperty('indeterminate')).toBe(true);
+      expect(element.shadowRoot.querySelector('input').indeterminate).to.be.true;
     });
   });
 
   it('should prevent scrolling on space bar press', async () => {
-    page = await newE2EPage();
-    await page.setContent(
-      `<div style="height: 100px; overflow: scroll" id="scroll-context">
-              <div style="height: 500px">
-                <sbb-checkbox></sbb-checkbox>
-              </div>
-            </div>`,
+    const root = await fixture(
+      html`<div style="height: 100px; overflow: scroll" id="scroll-context">
+        <div style="height: 500px">
+          <sbb-checkbox></sbb-checkbox>
+        </div>
+      </div>`,
     );
-    element = await page.find('sbb-checkbox');
-    expect(element).not.toHaveAttribute('checked');
-    expect(await page.evaluate(() => document.querySelector('#scroll-context').scrollTop)).toBe(0);
-
-    await element.press(' ');
-    await page.waitForChanges();
-
-    expect(element).toHaveAttribute('checked');
-    expect(await page.evaluate(() => document.querySelector('#scroll-context').scrollTop)).toBe(0);
+    element = root.querySelector('sbb-checkbox');
+    expect(element).not.to.have.attribute('checked');
+    await expect(root.scrollTop).to.be.equal(0);
+
+    element.focus();
+    await sendKeys({ press: ' ' });
+    await waitForLitRender(element);
+    await waitForCondition(() => element.hasAttribute('checked'));
+    await expect(root.scrollTop).to.be.equal(0);
```

Comment:
Intellij here mixes types some how (did the same first :D), but it's not async. Maybe we once can fix typings somehow, but for now you can remove await to be aligned with other code
```suggestion
    expect(root.scrollTop).to.be.equal(0);
```